config/esp32/toolchain/BUILD.gn

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2020 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import("//build_overrides/chip.gni")
+
+import("//build/toolchain/gcc_toolchain.gni")
+
+declare_args() {
+  ar_ext = ""
```

Comment:
If they're defined here, probably makes sense to name them something like

esp32_ar
esp32_cc
esp32_cxx

Reply:
fixed